cmd/snap/cmd_snap_op.go

```diff
@@ -940,10 +940,17 @@ func (x *cmdRefresh) holdRefreshes() (err error) {
 		return err
 	}
 
+	var timeStr string
+	if opts.Time == "forever" {
+		timeStr = "indefinitely"
+	} else {
+		timeStr = fmt.Sprintf("until %s", opts.Time)
```

Comment:
These should also get translated, I guess:

```suggestion
		timeStr = i18n.G("indefinitely")
	} else {
		timeStr = fmt.Sprintf(i18n.G("until %s"), opts.Time)
```

Reply:
Indeed, thank you